integration-tests/src/tests/client/features/delegate_action.rs

```diff
@@ -945,3 +952,100 @@ fn meta_tx_create_eth_implicit_account() {
     }
     meta_tx_create_implicit_account(eth_implicit_test_account());
 }
+
+// TODO(eth-implicit) Remove this test and replace it with tests that directly call the `Wallet Contract` when it is ready.
+/// Creating an ETH-implicit account with meta-transaction, then attempting to use it with another meta-transaction.
+///
+/// Depending on `rlp_transaction` blob that is sent to the `Wallet Contract`
+/// the transaction is either authorized or unauthorized.
+/// The parameter `authorized` controls which case will be tested.
+fn meta_tx_call_wallet_contract(authorized: bool) {
+    if !checked_feature!("stable", EthImplicitAccounts, PROTOCOL_VERSION) {
+        return;
+    }
+    let genesis = Genesis::test(vec![alice_account(), bob_account(), carol_account()], 3);
+    let relayer = alice_account();
+    let node = RuntimeNode::new_from_genesis(&relayer, genesis);
+    let sender = bob_account();
+
+    let secret_key = SecretKey::from_seed(KeyType::SECP256K1, "test");
+    let public_key = secret_key.public_key();
+    let eth_implicit_account = derive_eth_implicit_account_id(public_key.unwrap_as_secp256k1());
+    let other_public_key = SecretKey::from_seed(KeyType::SECP256K1, "test2").public_key();
+
+    // Although ETH-implicit account can be zero-balance, we pick 1 here in order to make transfer later from this account.
+    let transfer_amount = 1u128;
+    let actions = vec![Action::Transfer(TransferAction { deposit: transfer_amount })];
+    // Create ETH-implicit account by funding it.
+    node.user()
+        .meta_tx(sender.clone(), eth_implicit_account.clone(), relayer.clone(), actions)
+        .unwrap()
+        .assert_success();
+
+    let target = carol_account();
+    let initial_balance = node.view_balance(&target).expect("failed looking up balance");
+
+    // TODO(eth-implicit) Append appropriate values to the RLP stream when proper `Wallet Contract` is implemented.
+    let mut stream = RlpStream::new_list(3);
+    stream.append(&target.as_str());
+    // The RLP trait `Encodable` is not implemented for `u128`. We must encode it as bytes.
```

Comment:
I do not think such a PR will be necessary. When the real version of the Wallet Contract is in place the integration tests will not be trying to encode `u128` values directly (any Near balances will be upgraded to 256-bit values because that is the Ethereum standard).

Reply:
I've added a comment so that we do not forget about it.